tools/cleanup/cleanup.go

```diff
@@ -162,7 +162,7 @@ func (d* ImageDeleter) deleteImage(ref string) error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to parse reference %q", ref)
 	}
-	if err := remoteDelete(image, remote.WithAuthFromKeychain(auther)); err != nil {
+	if err := remoteDelete(image, remote.WithAuthFromKeychain(author)); err != nil {
```

Comment:
This is not author. :)

```suggestion
	if err := remoteDelete(image, remote.WithAuthFromKeychain(auther)); err != nil {
```